srcs/app/http/HttpStatusCodes.cpp

```diff
@@ -0,0 +1,64 @@
+#include <HttpStatusCodes.hpp>
+#include <utility>
+
+const HttpStatusCodes::StatusCodesMap_
+HttpStatusCodes::kStatusCodesMap_ = CreateStatusCodesMap_();
+
+const HttpStatusCodes::StatusCodesMap_
+HttpStatusCodes::CreateStatusCodesMap_() {
```

Comment:
I don't feel really comfortable by having the codes hardcoded in a class... Although is the best solution for now, that is for sure.

We could evaluate having a new task to move them to a external config file. Something like a `.yaml`. The only problem here would be reading from that file. Just thinking out loud, but it may not be possible because the limitation we have of being allowed to only use C++98

Reply:
I think it's a great idea, and I'm ok with either. nginx has the [MIME types](https://github.com/nginx/nginx/blob/master/conf/mime.types) in a config file and the [status codes](https://github.com/nginx/nginx/blob/master/src/http/ngx_http_special_response.c) hardcoded.